### ISSUE TITLE
A shift of the head line if a resource doesn't have the true delete policy

### ISSUE THREAD

**Issue by Brand3000** [outl1ne/nova-sortable#140]
If I don't allow the delete policy, the head line of the list has a shift on the left and right respectively.

**Comment by reppair** [comment]
Ah, I was about to report that one as well.

Basically if Nova does not render the checkboxes to select rows, there are more `td` elements per row than `th` elements in the `thead` of the table. Respectively the column headings do not match the data in the rows.

Here is an example
<img width="1395" alt="Screen Shot 2022-09-27 at 23 02 29" src="https://user-images.githubusercontent.com/10784954/192625389-9158077d-0914-4910-85b3-6ebcb9392975.png">

Steps to reproduce
1. On a sortable resource (which does not define any actions)
2. Prevent delete action to hide the selection checkboxes
```php
public function authorizedToDelete(Request $request)
{
    return false;
}
```

Even without the checkboxes, the expected result is proper alignment between `th` and `td' like so
<img width="1413" alt="Screen Shot 2022-09-27 at 23 02 13" src="https://user-images.githubusercontent.com/10784954/192626007-5e569c60-4911-44f6-8697-d40d50103d06.png">

**Comment by Tarpsvo** [comment]
Heya! @shaffe-fr made a PR which fixes this behaviour and it was released in version 3.3.0. Please try it out. Good luck!

**Comment by Brand3000** [comment]
It works fine on resources, but in relations the bug still persists.
